### PR TITLE
Convert rules to use guidebook parsing

### PR DESCRIPTION
**Pull request by EmoGarbage404**
## About the PR

`.txt` stinky `.xml` glorious

changes the rules windows to use guidebook parsing.
changes the cvar to now use a guidebook prototype.
its very cool works great

needs #28452

## Media

image

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes

**Changelog**

no cl no fun

### CODE REVIEW

File: Content.Client/Info/InfoSystem.cs [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
+using Robust.Shared.Prototypes;
 
 namespace Content.Client.Info;
 
 public sealed class InfoSystem : EntitySystem
 {
-    public RulesMessage Rules = new RulesMessage("Server Rules", "The server did not send any rules.");
-    [Dependency] private readonly RulesManager _rules = default!;
+    // TODO: don't merge with this
+    public ProtoId<GuideEntryPrototype> Rules = "SS!4";
```
Comment (AJCM-git) [inline Content.Client/Info/InfoSystem.cs:10]: a

File: Content.Client/Info/RulesControl.xaml [hunk 2]
```diff
@@ -1,6 +1,18 @@
-﻿<BoxContainer xmlns="https://spacestation14.io"
-              Name="InfoContainer"
-              Orientation="Vertical"
-              Margin="2 2 0 0"
-              SeparationOverride="10"
-              Access="Public"/>
+﻿<BoxContainer xmlns="https://spacestation14.io" Orientation="Vertical" HorizontalExpand="True" VerticalExpand="True" Access="Public">
```
Comment (AJCM-git) [inline Content.Client/Info/RulesControl.xaml:1]: not your fault but public access with no name

File: Content.Client/Info/RulesManager.cs [hunk 3]
```diff
@@ -90,16 +95,8 @@ private void OnAcceptPressed()
...
-        var rules = _sysMan.GetEntitySystem<InfoSystem>().Rules;
-        rulesSection.SetText(rules.Title, rules.Text, true);
-    }
-
-    public Control RulesSection()
-    {
-        rulesSection = new InfoSection("", "", false);
-        UpdateRules();
-        return rulesSection;
+        return _prototypeManager.Index(_sysMan.GetEntitySystem<InfoSystem>().Rules);
```
Comment (AJCM-git) [inline Content.Client/Info/RulesManager.cs:100]: Yeah, i see the reason of the TODO now (its prob something else but this SUCKS)

File: Content.Client/Info/InfoSystem.cs [hunk 4]
```diff
@@ -19,7 +20,6 @@ public override void Initialize()
     private void OnRulesReceived(RulesMessage message, EntitySessionEventArgs eventArgs)
     {
         Log.Debug("Received server rules.");
-        Rules = message;
-        _rules.UpdateRules();
+        Rules = message.Guide;
     }
```
Comment (AJCM-git) [inline Content.Client/Info/InfoSystem.cs:24]: can you subscribe to this in the UIController instead?

**Review by AJCM-git — CHANGES_REQUESTED** [review]
Holy moly jank like this is why UiControllers were made (tho i dont think this predates them so wtf?)

**Comment by EmoGarbage404** [comment]
now configured to use the prototypes from the rules PR

**Review by AJCM-git — APPROVED** [review]

**Pull request merged**